openapi.yaml

```diff
@@ -194,6 +194,182 @@ paths:
           $ref: "#/components/responses/Unauthorized"
         "504":
           $ref: "#/components/responses/CantConnectToDatabase"
+  /courses:
+    summary: Perform Create and Read operations on courses.
+    description: This resource comprises of READ and Write operations that can be performed on the courses object.
+    get:
+      summary: Get all the courses.
+      description: Returns a list of course.
+      parameters:
+        - in: query
+          name: page
+          description: Page number of results to retrieve.
+          schema:
+            type: integer
+            default: 1
+        - in: query
+          name: limit
+          description: Number of results to retrieve per page.
+          schema:
+            type: integer
+            minimum: 1
+            maximum: 50
+            default: 10
+      tags:
+        - Courses
```

Comment:
If we're going to add tags like this *(and we should!)* then let's add them to the other paths as well.